examples/js/exporters/GLTFExporter.js

```diff
@@ -180,6 +180,27 @@ THREE.GLTFExporter.prototype = {
 
 		}
 
+		/**
+		 * Get the required size + padding for a buffer, rounded to the next 4-byte boundary.
+		 * https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#data-alignment
+		 *
+		 * @param {Integer} bufferSize The size the original buffer.
+		 * @returns {Integer} new buffer size with required padding.
+		 *
+		 */
+		function getPaddedBufferSize( bufferSize ) {
+			
+			// adapted from https://github.com/AnalyticalGraphicsInc/obj2gltf/blob/master/lib/getBufferPadded.js
+
+			var boundary = 4;
+
+			var remainder = bufferSize % boundary;
+			
+			var padding =  (remainder === 0 ) ? 0 : boundary - remainder;
```

Comment:
nit: could you remove the parens here, just: `remainder === 0 ? 0 : boundary - remainder;`